app/javascript/components/App.js

```diff
@@ -14,14 +14,33 @@ import {
 } from 'react-router-dom'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+
+componentDidMount(){
+  this.readAparment()
+}
+
+readApartment = () => {
+  fetch ("http://localhost:3000/apartments")
```

Comment:
Since this is a monolithic app, you should remove everything before "/apartments"